AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/DataStoreHubEventsTests.swift

```diff
@@ -27,6 +27,9 @@ class DataStoreHubEventTests: HubEventsIntegrationTestBase {
     ///    - subscriptionEstablished received, payload should be nil
     ///    - syncQueriesStarted received, payload should be: {models: ["Post", "Comment"]}
     ///    - outboxStatus received, payload should be {isEmpty: true}
+    ///    - modelSynced received, payload should be:
+    ///      {modelName: "Some Model name", isFullSync: true/false, isDeltaSync: false/true, createCount: #, updateCount: #, deleteCount: #}
```

Comment:
this comment line is pass the max column width for swiftlint/swiftformat rules? i wonder if this target isn't set up properly